src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglx_utils.py

```diff
@@ -12,12 +12,15 @@ def add_recording_extractor_properties(recording_extractor) -> None:
     probe = recording_extractor.get_probe()
     channel_ids = recording_extractor.get_channel_ids()
 
+    # Should follow pattern 'Imec0', 'Imec1', etc.
+    probe_name = recording_extractor.stream_id[:5].capitalize()
+
     if probe.get_shank_count() > 1:
-        group_name = [contact_id.split("e")[0] for contact_id in probe.contact_ids]
         shank_electrode_number = [int(contact_id.split("e")[1]) for contact_id in probe.contact_ids]
+        group_name = [f"{probe_name}{contact_id.split('e')[0]}" for contact_id in probe.contact_ids]
```

Comment:
@h-mayorquin Inspired by your simpler strategy of going down the utils here so it applies to all cases not just the converter pipe

Reply:
Sounds good. I think we should not add shank information in shankless probes but I will refactor this after we fix this anyway so this is OK..